src/main/java/com/softserveinc/task03/Country.java

```diff
@@ -0,0 +1,97 @@
+package com.softserveinc.task03;
+
+import com.softserveinc.task02.City;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Country {
+    private String name;
+    private String continent;
+    private String code;
+    private City capital;
+    private List<City> cities;
+
+    public Country() {
+
+    }
+
+    public Country(String name, String continent, String code, City capital) {
+        this.name = name;
+        this.continent = continent;
+        this.code = code;
+        this.capital = capital;
+        this.cities = new ArrayList<City>();
```

Comment:
ArrayList is OK, but why you don't use the array as described in the task?

Reply:
Because ArrayList is a variable-length array and does not require a setting up of length.
I'll try to use a simple array.